LICENSE

```diff
@@ -200,3 +200,43 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+SUBCOMPONENTS:
+
+The Daffodil VS Code Debug project contains subcomponents with separate copyright
```

Comment:
It's probably best to refer to this as `Apache Daffodil VS Code Debug`. I believe the general rule is the first time the project name comes up it should be `Apache Daffodil` and all following occurrences can just be `Daffodil`. Apache is pretty protective of their trademark and wants to ensure Apache is included where appropriate.

Reply:
Addressed this